RestartManager.js

```diff
@@ -3,41 +3,54 @@ const NativeCodePush = require("react-native").NativeModules.CodePush;
 
 const RestartManager = (() => {
     let _allowed = true;
-    let _restartPending = false;
+    let _restartInProgress = false;
+    let _restartQueue = [];
```

Comment:
For my own education - is there a reason we'd want to restart multiple times back-to-back, as opposed to squashing them into a single restart?